controller_manager/src/controller_manager.cpp

```diff
@@ -499,7 +499,10 @@ void ControllerManager::init_controller_manager()
   // Get parameters needed for RT "update" loop to work
   if (is_resource_manager_initialized())
   {
-    resource_manager_->import_joint_limiters(robot_description_);
+    if (enforce_command_limits_)
+    {
+      resource_manager_->import_joint_limiters(robot_description_);
+    }
```

Comment:
do we need this at all here, if we import that in the `init_resource_manager`?

Reply:
or different question,  when `is_resource_manager_initialized` here true?
And in case it is, and we receive a robot_description afterwards: Will the joint limits be added twice? I don't see a reset of `joint_limiters_interface_` or `limiters_data_`?

Reply:
No they don't. They use the insert command, so they don't get updated. The implementation is inside the Resource Manager

Reply:
ah true, this is a map.